server/common/aws_secret_utils.py

```diff
@@ -49,6 +54,18 @@ def handle_config_from_secret(app_config):
             logging.error(f"set {attr} from secret")
             app_config.update_server_config(**{attr : val})
 
+    # update default dataset configuration attributes
+    for key, attr in default_dataset_attrs:
+        cur_val = getattr(app_config.default_dataset_config, attr)
+        if cur_val:
+            continue
+
+        # replace the attr with the secret if it is not set
+        val = secrets.get(key)
+        if val:
+            logging.error(f"set {attr} from secret")
```

Comment:
I'm not sure why I had this as logging.error.  It should be "info".  It is definitely not an error.

Reply:
ok cool, will update